### PR TITLE
#338 unit test circle

### PR DESCRIPTION
**Pull request by matt-miller-rgi**

### CODE REVIEW

File: sdk/sdk-api/src/main/java/mil/emp3/api/Circle.java [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 import org.cmapi.primitives.IGeoPosition;
 
 import java.util.List;
 
+import mil.emp3.api.abstracts.Feature;
 import mil.emp3.api.enums.FeatureTypeEnum;
 import mil.emp3.api.interfaces.IEmpBoundingBox;
 import mil.emp3.api.utils.EmpBoundingBox;
-import mil.emp3.api.utils.GeographicLib;
+import mil.emp3.api.utils.GeoLibrary;
```
Comment (stevendlander) [inline sdk/sdk-api/src/main/java/mil/emp3/api/Circle.java]: GeoLibrary should be deprecated, you will need to use GeographicLib for this class.

File: sdk/sdk-api/src/main/java/mil/emp3/api/Circle.java [hunk 2]
```diff
@@ -55,11 +49,8 @@ public Circle(IGeoCircle renderable) {
         if(null == renderable) {
             throw new IllegalArgumentException("Encapsulated GeoCircle must be non-null");
         }
-        this.setRadius(makePositive(this.getRadius(), "Invalid radius. NaN"));
-
-        if (this.getRadius() < MINIMUM_RADIUS) {
-            throw new IllegalArgumentException("Invalid radius. " + this.getRadius() + " Minimum supported " + MINIMUM_RADIUS);
-        }
+        //seeting renderable bypasses radius validation
```
Comment (stevendlander) [inline sdk/sdk-api/src/main/java/mil/emp3/api/Circle.java]: Seeting?

**Review by stevendlander — CHANGES_REQUESTED** [review]

File: sdk/sdk-view/src/test/java/mil/emp3/api/CircleTest.java [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void zeroSetRadius() {
+        circ = new Circle();
+        circ.setRadius(0.0);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void NaNSetRadius() {
```
Comment (JACochran) [inline sdk/sdk-view/src/test/java/mil/emp3/api/CircleTest.java]: all methods should be camel-cased so the "N" should be lowercase

File: sdk/sdk-view/src/test/java/mil/emp3/api/CircleTest.java [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+        circ = new Circle(-45.5);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void zeroRadiusConstructor() {
+        circ = new Circle(0.0);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void NaNRadiusConstructor() {
```
Comment (JACochran) [inline sdk/sdk-view/src/test/java/mil/emp3/api/CircleTest.java]: N should be lowercase

File: sdk/sdk-view/src/test/java/mil/emp3/api/CircleTest.java [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+                new HashMap());
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void nullGeoCircleConstructor() {
+        circ = new Circle(null);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void NegativeRadiusConstructor() {
```
Comment (JACochran) [inline sdk/sdk-view/src/test/java/mil/emp3/api/CircleTest.java]: N should be lowercase

**Review by JACochran — CHANGES_REQUESTED** [review]

**Review by stevendlander — APPROVED** [review]

**Review by JACochran — APPROVED** [review]

**Pull request merged**